tianshou/data/buffer.py

```diff
@@ -104,27 +104,27 @@ class ReplayBuffer:
 
     :param int size: the size of replay buffer.
     :param int stack_num: the frame-stack sampling argument, should be greater
-        than 1, defaults to 0 (no stacking).
+        than or equal to 1, defaults to 1 (no stacking).
```

Comment:
I’m not sure it is mentioned in the tutorials. Could you please have a double-check?

Reply:
According to [this](https://github.com/thu-ml/tianshou/search?q=stack_num&unscoped_q=stack_num), no.